sympy/matrices/expressions/sets.py

```diff
@@ -60,3 +62,6 @@ def _check_dim(cls, dim):
             raise ValueError(
                 "The dimension specification {} should be "
                 "a nonnegative integer.".format(dim))
+
+    def _kind(self):
+        return SetKind(MatrixKind(NumberKind))
```

Comment:
There should be two blank lines before this function. Top-level elements should be separated by two blank lines